crates/rpc/rpc-types/src/eth/trace/parity.rs

```diff
@@ -267,14 +267,18 @@ pub struct LocalizedTransactionTrace {
     /// Hash of the block, if not pending
     ///
     /// Note: this deviates from <https://openethereum.github.io/JSONRPC-trace-module#trace_transaction> which always returns a block number
+    #[serde(skip_serializing_if = "Option::is_none")]
     pub block_hash: Option<H256>,
     /// Block number the transaction is included in, None if pending.
     ///
     /// Note: this deviates from <https://openethereum.github.io/JSONRPC-trace-module#trace_transaction> which always returns a block number
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
I'm not sure, so you should double check, but I don't think the block hash (above) or the block number (here) would ever be none, so this might be confusing to someone reading in the future.